envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/SnapshotProperties.kt

```diff
@@ -66,6 +66,7 @@ typealias Client = String
 class IncomingPermissionsProperties {
     var enabled = false
     var clientIdentityHeaders = listOf("x-service-name")
+    var xClientNameTrustedHeader = "x-client-name-trusted"
```

Comment:
I think we should add an ability to disable lua client identity header filter here. For example by settings this value to null. We need a way to disable this feature. It will be needed if for example not all envoys are upgraded to supported version yet and we don't want to block other changes in EC.